docs/configuration.md

```diff
@@ -41,6 +41,7 @@ Key | Description | Type | Default | Required
 `support_email` | Allow to define support email | Type | Default | Required
 `resellerId` | The resellerId key is the accountId of your master account, and is needed for some reseller features. For example it won't prompt for a credit card the sub-accounts that have a different resellerId than this resellerId | `String` | | `false`
 `whitelabel` | Contains all the flags that are whitelabel-able via the Branding app. Setting them in the config file will set the defaults if you don't use any whitelabel. If the domain used is defined in the whitelabel database, we'll override the following settings by what is set in the whitelabel document. | `Object`([#whitelabel](#whitelabel)) | | `true`
+`extraDevices` | Contains names of additional devices integrations. | `Array` | | `false`
```

Comment:
How about `extraDeviceTypes` to delineate that it will contain types of devices (e.g.  values for `device_type`)?

```suggestion
`extraDevices` | Contains types of additional devices corresponding to integrations. | `Array` | | `false`
```

Reply:
Thanks! Because the idea of this new key is that it will be set once the TeamMate integration has been configured for the cluster, and will allow the user to create a new teammate device, I also updated its name to `allowedExtraDeviceTypes`.